docs/DeveloperGuide.md

```diff
@@ -162,66 +162,64 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
-### Delete/deletetask feature
+### Add Task feature
 
 #### Implementation
 
-The implementation of the delete/deletetask function is facilitated by `LogicManager`. It takes user input as 2 arguments, the command word being either `delete` or `deletetask`, depending on whether the user wants to delete a person or task respectively. The second argument is an index, denoting the index of the person or task the user wishes to delete.
+The implementation of the add task function is facilitated by `LogicManager`. It takes user input as 2 arguments, the command word being "todo". The second argument is a description of the task.
 
 The format is as follows:
-- `delete {Person_ID}`
-- `deletetask {TASK_ID}`
+- `todo task/{Task_Description}`
```

Comment:
do we still wanna keep curly brackets?

Reply:
I haven't really done the DG this was just a reordering of the section.